graalvm/build.gradle

```diff
@@ -54,7 +54,7 @@ task buildGraal() {
     def excludeDynamicLanguages = findProperty('excludeDynamicLanguages')
 
     // Register graal repository dummy file as an input and repository directory as output
-    inputs.files(graalCloneDummy)
+    inputs.files(graalPatchDummy)
```

Comment:
Can you leave a comment explaining why we cannot monitor the actual files that are used as input for this task?